src/constants.js

```diff
@@ -34,13 +40,26 @@ export function decrementActiveDropZoneCount() {
  */
 export function overrideItemIdKeyNameBeforeInitialisingDndZones(newKeyName) {
     if (activeDndZoneCount > 0) {
-        throw new Error("can only override the id key before initialising any dndzone");
+        throw new Error(
+            'can only override the id key before initialising any dndzone'
+        );
     }
-    if (typeof newKeyName !== "string") {
-        throw new Error("item id key has to be a string");
+    if (typeof newKeyName !== 'string') {
+        throw new Error('item id key has to be a string');
     }
-    console.debug("overriding item id key name", newKeyName)
+    printDebug(() => 'overriding item id key name', newKeyName);
```

Comment:
what is going to happen to the second parameter in such cases (`newKeyItem`)?

Reply:
Sorry I did not see these. I changed thes multi-argument outputs to return arrays now. Any other idea?